docs/source/conf.py

```diff
@@ -28,5 +28,4 @@
 
 html_theme = "alabaster"
 
-# Example configuration for intersphinx: refer to the Python standard library.
-intersphinx_mapping = {"http://docs.python.org/": None}
+intersphinx_mapping = {"python": ("http://docs.python.org/3/", None)}
```

Comment:
Taken from Iris:

https://github.com/SciTools/iris/blob/38a8ab9f0ecedc481e74d74553bf85b93e8b0603/docs/src/conf.py#L257

Reply:
Os the intersphinx_mapping actually generating an cross links to numpy? I couldn't find any in the generated docs.
I assume this is just a fix to stop it from breaking the docs build?

Reply:
@ukmo-ccbunney this mapping is for Python, rather than NumPy. You're right that no cross-links are currently being rendered in the docs. The only usage is `approx_equal`, which is both un-documented and deprecated:

https://github.com/SciTools/cf-units/blob/65e545c971f96eeddde9fdb45386838756410288/cf_units/util.py#L15-L21

... but it does show that there is precedent for using a Python intersphinx so I would be reluctant to remove it in case it's useful to future developers.

Removing intersphinx entirely would not break the docs build, if you felt strongly that we should do that.

Reply:
Apologies - I meant Python (I saw numpy in the Iris example and typed that instead)
Happy to leave it - perhaps we'll add some more intersphinx links in the future (like for numpy!)